multires-lyrics-search/flows/index.yml

```diff
@@ -8,16 +8,17 @@ pods:
   - name: encoder
     show_exc_info: true
     uses: pods/encode.yml
-    parallel: $JINA_PARALLEL
+    polling: any
+    shards: $JINA_PARALLEL
```

Comment:
use `JINA_SHARDS` naming

Reply:
same reason as in cxmodal, next iteration, I have it only on the `os env` part and I would like to keep the code in sync with docker image